src/AutoCrudAdmin/TagHelpers/FormInputTagHelper.cs

```diff
@@ -93,9 +111,9 @@ public override Task ProcessAsync(TagHelperContext context, TagHelperOutput outp
             {
                 this.PrepareMultiChoiceCheckbox(output);
             }
-            else if (this.FormControlType == FormControlType.ExpandableMultiChoiceCheckBox)
```

Comment:
Why is this removed completely? Isn't used for different purpose? Don't we want them both, or the autocomplete is just improved version of the ExpandableMultiChoiceCheckBox?

Reply:
I am not sure how that happened, but I reversed these changes. These are separate needed form controls.